src/psyclone/tests/nemo/nemo_psy_test.py

```diff
@@ -116,17 +116,13 @@ def test_array_valued_function():
 
 
 def test_do_while():
```

Comment:
I think we can/should just remove this test now as the functionality should be tested elsewhere.